docs/api/commands/exec.md

```diff
@@ -44,7 +45,9 @@ This will create a new Bash session in the container `ubuntu_bash`.
 
 ## Divergence
 
-There is no known divergence between the Triton SDC Docker and Docker Inc. implementations of this method. Please contact Joyent support or file a ticket if you discover any.
+There is no known divergence between the Triton SDC Docker and Docker Inc.
```

Comment:
Sorry, I'm a little understanding. "--privileged" has been removed from the upstream documentation for this command, see https://github.com/docker/docker/blob/master/docs/sources/reference/commandline/cli.md#exec